markdown_it/renderer.py

```diff
@@ -6,14 +6,34 @@ class Renderer
 rules if you create plugin and adds new token types.
 """
 import inspect
-from typing import MutableMapping, Optional, Sequence
+import sys
+from typing import (
+    ClassVar,
+    MutableMapping,
+    Optional,
+    Sequence,
+)
 
 from .common.utils import unescapeAll, escapeHtml
 from .token import Token
 from .utils import OptionsDict
 
+if sys.version_info < (3, 8):
+    from typing_extensions import Protocol
+else:
+    from typing import Protocol
```

Comment:
```suggestion
try:
	from typing import Protocol
except ImportError:
    from typing_extensions import Protocol    
```

I've literally just has an issue where `sys.version_info` was not reliable

Reply:
Changed. Please let me know what the issue was. I'm very interested.

Reply:
It actually turned out to be an issue with importing of `importlib_metadata` instead of `importlib-metadata` in a Conda environment 😒 (see https://github.com/python/importlib_metadata/issues/308 and executablebooks/MyST-NB#321), which actually illustrates my "worry" in https://github.com/executablebooks/markdown-it-py/pull/154#issuecomment-811740183

But anyhow, I think this approach is still slightly "safer"

Reply:
Ah, so conda not doing the name normalization if I understand correctly? Good to know, thanks! I don't use Conda so no need to change any of my personal projects :+1:

Reply:
yeh seems to be the case 😬

Reply:
despite that it is generally very good; particularly in resolving dependencies and/or if you are using non-python dependencies